functorch/_src/aot_autograd.py

```diff
@@ -472,12 +472,15 @@ def forward(ctx, *deduped_flat_tensor_args):
         def backward(ctx, *flat_args):
             contiguous_args = [t.contiguous() if torch.is_tensor(t) else t for t in flat_args]
             all_args = list(ctx.symints) + list(ctx.saved_tensors) + list(contiguous_args)
+            del contiguous_args
             if CompiledFunction.compiled_bw is None:
+                # TODO - pass in fake tensors ?
```

Comment:
this wasn't actually necessary for memory savings and had a bad interaction with the pr above this + didn't repro with minifier - deferring on passing in fake for now